TESTING.rst

```diff
@@ -621,6 +726,53 @@ print output generated test logs and print statements to the terminal immediatel
 You can modify the tests or KubernetesPodOperator and re-run them without re-deploying
 airflow to KinD cluster.
 
+
+Sometimes there are side effects from running tests. You can run ``redeploy_airflow.sh`` without
+recreating the whole cluster. This will delete the whole namespace, including the database data
+and start a new Airflow in the cluster.
+
+.. code-block:: bash
+
+    ./scripts/ci/redeploy_airflow.sh
+
+If needed you can also delete the cluster manually:
+
+
+.. code-block:: bash
+
+    kind get clusters
+    kind delete clusters <NAME_OF_THE_CLUSTER>
+
+Kind has also useful commands to inspect your running cluster:
+
+.. code-block:: text
+
+    kind --help
+
+    Usage:
+      kind [command]
+
+    Available Commands:
+      build       Build one of [node-image]
+      completion  Output shell completion code for the specified shell (bash, zsh or fish)
+      create      Creates one of [cluster]
+      delete      Deletes one of [cluster]
+      export      Exports one of [kubeconfig, logs]
+      get         Gets one of [clusters, nodes, kubeconfig]
+      help        Help about any command
+      load        Loads images into nodes
+      version     Prints the kind CLI version
+
+    Flags:
+      -h, --help              help for kind
+          --loglevel string   DEPRECATED: see -v instead
+      -q, --quiet             silence all stderr output
+      -v, --verbosity int32   info log verbosity
+          --version           version for kind
```

Comment:
I think we can just say, run ``kind --help`` otherwise we will need to keep this up-to-date with changes in kind cli